gallery/visualtests/pageLayouts.js

```diff
@@ -23,11 +23,11 @@ describe('Page Layout', function() {
       it('looks right', async function() {
         await disableLoadingSpinnerAnimation();
         let screenshot = await getPage().screenshot();
-        expect(screenshot).toMatchImageSnapshot();
+        expect(Buffer.from(screenshot)).toMatchImageSnapshot();
```

Comment:
Just for my understanding, why is this change needed?

Reply:
> Just for my understanding, why is this change needed?

`page.screenshot()` now returns an `Uint8Array` instead of a buffer.  Passing that directly to `toMatchImageSnapshot` didn't parse the PNG correctly and resulted in errors.  Googling suggested this fix.